src/reducers/index.js

```diff
@@ -45,6 +45,12 @@ const initialState = {
   // Height of the best chain of the network arrived from ws data
   height: 0,
   wallet: null,
+  // Metadata of tokens
+  tokenMetadata: {},
+  // Token list of uids that had errors when loading metadata
+  tokenMetadataErrors: [],
```

Comment:
It seems we are not using this piece of information yet, right?

Reply:
As we discussed privately we won't use this state for now